src/organism.jl

```diff
@@ -0,0 +1,24 @@
+
+
+Base.@kwdef struct OrganismalPars{F}
+    α_org_dorsal::F = Param(0.8, bounds=(0.2, 1.0))
+    α_org_ventral::F = Param(0.8, bounds=(0.2, 1.0))
+    ϵ_org::F = Param(0.95, bounds=(0.1, 0.0))
```

Comment:
I think we should distinguish dorsal and ventral for this too, in retrospect

Reply:
OK!